lib/sles4sap/ipaddr2.pm

```diff
@@ -60,11 +71,30 @@ our $nat_pub_ip = DEPLOY_PREFIX . '-nat_pub_ip';
 # Storage account name must be between 3 and 24 characters in length
 # and use numbers and lower-case letters only.
 our $storage_account = DEPLOY_PREFIX . 'storageaccount';
-our $priv_ip_range = '192.168.';
-our $frontend_ip = $priv_ip_range . '0.50';
+our %priv_net_address_range = get_private_ip_range();
```

Comment:
This code (that even have external dependency to qesapdeploy)  is executed implicitly at some point (before the test start?). Is it possible to avoid function call in this scope?